forms/registration.go

```diff
@@ -18,17 +18,11 @@ type RegistrationForm struct {
 func (rf *RegistrationForm) Valid(v *validation.Validation) {
 	userExist := models.UserEmailAlreadyExist(rf.Email)
 	if userExist {
-		validationError := v.SetError("Email", "User with this email already exist")
-		if validationError == nil {
-			logs.Info("Failed to set error on validation")
-		}
+		_ = v.SetError("Email", constants.UserAlreadyExist)
```

Comment:
Shall we return early? and may be return single error from this service as I think in the flash error, it is showing only single error. 

```go
_ = v.SetError("Email", constants.UserAlreadyExist)
return
```

Reply:
✅  done on [faded5f](https://github.com/carryall/go-google-scraper-challenge/pull/69/commits/faded5fe1f9e7ccf51fd30982bddd8e5a984caca)